src/assets/css/global.css

```diff
@@ -0,0 +1,13 @@
+body {
+  background-color: #fff;
+  color: #2c2c2c!important
```

Comment:
Please add space between `#2c2c2c` and `!important`

Reply:
Pushed the commit. thanks.